chapters/0-introduction.tex

```diff
@@ -156,15 +156,12 @@ \section{How to read this book}
 that contain all the code, data, and meta-information needed
 for others to evaluate and reproduce your work.
 
-\begin{fullwidth}
-\begin{figure}
-	\centering
-	\includegraphics[width=1.25\linewidth]{diagrams/Intro}
-	\caption{}
-	\label{fig:intro}
-\end{figure}
-\end{fullwidth}
-
+Figure \ref{fig:intro} shows a summary of the hands-on data tasks 
```

Comment:
The ref number does not show up for me:

![image](https://user-images.githubusercontent.com/15911801/92816552-aad43f00-f393-11ea-9d62-5234596086ef.png)

Reply:
I would cut figure numbering and refs entirely, just call them by name?

Reply:
done